src/components/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { ComponentMeta } from "@storybook/react";
+import { sprinkles } from "~/theme";
+import { Button } from "../Button";
+import { Text } from "../Text";
+import { ConfigurationIcon } from "../Icons";
+import { Toggle } from "../Toggle";
+import { Box } from "../Box";
+import { Popover } from ".";
+
+export default {
+  component: Popover,
+} as ComponentMeta<typeof Popover>;
+
+export const Default = () => {
+  return (
+    <Popover>
+      <Popover.Trigger asChild>
+        <Button fixedWidth variant="secondary">
+          <ConfigurationIcon />
+        </Button>
+      </Popover.Trigger>
+      <Popover.Content
+        className={sprinkles({
+          padding: 5,
+          backgroundColor: "subdued",
+        })}
+      >
+        <Popover.Arrow />
```

Comment:
question: why I as a user must care about putting arrow? Should be hidden in the Content component?